library/src/lib/menu/menu.component.ts

```diff
@@ -12,9 +15,24 @@ import {
     encapsulation: ViewEncapsulation.None,
     styleUrls: ['menu.component.scss']
 })
-export class MenuComponent {
+export class MenuComponent extends AbstractFdNgxClass {
```

Comment:
Could you please use `@HostBinding` along with `@Input` to bind class to input variable? There are some examples in code. There is no sense to use `AbstractFdNgxClass` anymore.

Reply:
done